packages/shared-ux/link/redirect_app/impl/src/redirect_app_links.styles.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const RedirectAppLinksStyles = () => {
```

Comment:
nIt: this should not be capitalized, which is reserved for React components and classes.  For example, note how EUI names their styles functions:

https://github.com/elastic/eui/blob/main/src/components/badge/badge.styles.ts#L21